lib/utils.js

```diff
@@ -37,16 +37,20 @@ export const uuidv4 = () => {
   });
 };
 
-export const formatDate = (timestamp) => {
+export const formatDate = (timestamp, includeTime = true) => {
   const date = new Date(timestamp * 1000);
-  const hours = date.getHours();
-  const minutes = date.getMinutes();
-  const ampm = hours >= 12 ? 'PM' : 'AM';
-  const formattedHours = hours % 12 || 12;
-
   const day = date.getDate();
   const month = date.toLocaleString('default', { month: 'short' });
   const year = date.getFullYear();
 
-  return `${formattedHours}:${minutes} ${ampm} | ${day} ${month}, ${year}`;
+  if (includeTime) {
```

Comment:
create a separate function for `formatDateTime`  and use `formatDate` function to get the date and do the additional time related code in the `formatDateTime` function.